android/src/main/java/com/geektime/rnonesignalandroid/RNOneSignal.java

```diff
@@ -538,6 +538,7 @@ public void sendOutcome(final String name, final Callback callback) {
          public void onSuccess(OSOutcomeEvent outcomeEvent) {
             if (outcomeEvent == null)
                callback.invoke(new WritableNativeMap());
+               Log.e("OneSignal", "sendOutcome OSOutcomeEvent is null");
```

Comment:
Maybe this can be called before the https://github.com/OneSignal/OneSignal-Android-SDK/pull/1248 in that way it appears in order? Also, we might want to use OneSignal Log?

Reply:
I ended up refactoring it based on a conversation with Josh regarding it matching iOS. We are no longer invoking the callback in the null case.

For the log, we don't use the native logging method anywhere in the bridge. Any change to that should probably be a stand alone PR